src/gameplay/glicko2.ts

```diff
@@ -0,0 +1,157 @@
+import type { Player, Match } from './types';
+
+class Glicko2 {
+  #epsilon: number;
+  #tau: number;
+
+  constructor() {
+    this.#epsilon = 0.000001;
+
+    // system constant tau is set as 0.5
+    this.#tau = 0.5;
+  }
+
+  #computeG(phi: number): number {
+    return 1 / Math.sqrt(1 + (3 * phi ** 2) / Math.PI ** 2);
+  }
+
+  #computeE(mu: number, mu_j: number, phi_j: number): number {
+    return 1 / (1 + Math.exp(-1 * this.#computeG(phi_j) * (mu - mu_j)));
+  }
+
+  #computeNewVolatility(
```

Comment:
Can you add `private` to the internal methods? 

It'll make it clearer what the entrypoint is.